ui-v2/app/styles/components/main-nav-horizontal/layout.scss

```diff
@@ -1,50 +1,73 @@
-%main-nav-horizontal > ul > li > [type='checkbox'] ~ div {
-  display: none;
-}
-%main-nav-horizontal > ul > li > [type='checkbox']:checked ~ div {
-  display: block;
-}
-%main-nav-horizontal [type='checkbox'] + label > * {
+%main-nav-horizontal .popover-menu > label > * {
   /* less space as the chevron adds space */
   padding-right: 4px !important;
 }
-%main-nav-horizontal > ul > li {
-  position: relative;
-}
 %main-nav-horizontal-action {
   display: block;
   padding: 5px 12px;
   white-space: nowrap;
 }
-%main-nav-horizontal-drop-nav {
+%main-nav-horizontal-menu-panel {
```

Comment:
Back back in the days before popover-menu the datacenter dropdown was referred to as the drop-nav. Fast forwards to today and the same thing is now the `%menu-panel` component (the panel that dropsdown when you click the trigger), so we updated the naming here to make it a bit more consistent and provide a better clue/comment as to what it refers to.